packages/react-next/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -17,135 +18,124 @@ const LARGE_IMAGE_SIZE = 71;
-    const { onFocus } = this.props;
-    if (onFocus) {
-      onFocus(evt, this.props);
-    }
   };
 
-  private _onRenderField = (props: IChoiceGroupOptionProps): JSX.Element => {
-    const { id, imageSrc, imageAlt = '', selectedImageSrc, iconProps } = props;
+  const renderField = (): JSX.Element => {
+    const { imageAlt = '', selectedImageSrc } = props;
 
-    const imageSize = props.imageSize ? props.imageSize : { width: 32, height: 32 };
+    const currentImageSize = props.imageSize ? props.imageSize : { width: 32, height: 32 };
 
-    const onRenderLabel = props.onRenderLabel
-      ? composeRenderFunction(props.onRenderLabel, this._onRenderLabel)
-      : this._onRenderLabel;
+    const renderedLabel = props.onRenderLabel
+      ? composeRenderFunction(props.onRenderLabel, onRenderLabel)
+      : onRenderLabel;
 
-    const label = onRenderLabel(props);
+    const label = renderedLabel(props);
 
     return (
-      <label htmlFor={id} className={this._classNames.field}>
+      <label htmlFor={id} className={classNames.field}>
         {imageSrc && (
-          <div className={this._classNames.innerField}>
-            <div className={this._classNames.imageWrapper}>
-              <Image src={imageSrc} alt={imageAlt} width={imageSize.width} height={imageSize.height} />
+          <div className={classNames.innerField}>
+            <div className={classNames.imageWrapper}>
+              <Image src={imageSrc} alt={imageAlt} width={currentImageSize.width} height={currentImageSize.height} />
             </div>
-            <div className={this._classNames.selectedImageWrapper}>
-              <Image src={selectedImageSrc} alt={imageAlt} width={imageSize.width} height={imageSize.height} />
+            <div className={classNames.selectedImageWrapper}>
+              <Image
+                src={selectedImageSrc}
+                alt={imageAlt}
+                width={currentImageSize.width}
+                height={currentImageSize.height}
+              />
             </div>
           </div>
         )}
         {iconProps && (
-          <div className={this._classNames.innerField}>
-            <div className={this._classNames.iconWrapper}>
+          <div className={classNames.innerField}>
+            <div className={classNames.iconWrapper}>
               <Icon {...iconProps} />
             </div>
           </div>
         )}
-        {imageSrc || iconProps ? <div className={this._classNames.labelWrapper}>{label}</div> : label}
+        {imageSrc || iconProps ? <div className={classNames.labelWrapper}>{label}</div> : label}
       </label>
     );
   };
 
-  private _onRenderLabel = (props: IChoiceGroupOptionProps): JSX.Element => {
-    return (
-      <span id={props.labelId} className="ms-ChoiceFieldLabel">
-        {props.text}
-      </span>
-    );
-  };
-}
+  const { onRenderField = renderField } = props;
+
+  const onChange = useConstCallback((evt: React.FormEvent<HTMLInputElement>): void => {
```

Comment:
we should remove `useConstCallback` from `onChange` `onBlur` and `onFocus` handlers. I don't see benefits for using them

Reply:
This one is a bit tricky--I do think if we're passing callbacks to our own components, it's a good idea to memoize, but it's probably not necessary when passing to native elements. The other tricky part is that `jsx-no-bind` lint rule can't distinguish between the two scenarios, so the options are either to disable it everywhere callbacks are passed to native elements (which could lead to people thinking it's okay to disable in other places where it's potentially more important), or memoize when technically not necessary.